src/main/java/com/conveyal/gtfs/loader/JdbcGtfsLoader.java

```diff
@@ -485,6 +476,53 @@ private int loadInternal (Table table) throws Exception {
         return numberOfRecordsLoaded;
     }
 
+    /**
+     * Returns the index of the specified field name for the array of fields provided.
+     * @param fields array of fields (intended to be derived from the headers of a csv text file)
+     * @param fieldName string name of the field to find
+     */
+    public static int getFieldIndex(Field[] fields, String fieldName) {
```

Comment:
Yeah, seeing how these are called from https://github.com/conveyal/datatools-server/pull/186/files#diff-b601a83ada6de294158d19894ceefe57, I think it might make more sense to put these methods in the Table class.

Reply:
Actually, this one doesn't have a Table argument, so maybe this should get put into the Field class.